src/gui/MainApplication.cpp

```diff
@@ -86,37 +86,36 @@ bool MainApplication::event(QEvent* event)
 }
 
 #ifdef LMMS_BUILD_WIN32
-// This can be moved into nativeEventFilter once Qt4 support has been dropped
-bool MainApplication::winEventFilter(MSG* msg, long* result)
+bool MainApplication::nativeEventFilter(const QByteArray& eventType,
+					void* message, long* result)
```

Comment:
Line is <100 characters, should be fine as single line
```suggestion
bool MainApplication::nativeEventFilter(const QByteArray& eventType, void* message, long* result)
```